src/NexusMods.App.UI/Overlays/MetricsOptIn/MetricsOptInViewModel.cs

```diff
@@ -1,28 +1,44 @@
 ﻿using System.Windows.Input;
+using JetBrains.Annotations;
 using NexusMods.Abstractions.Settings;
 using NexusMods.Abstractions.Telemetry;
-using NexusMods.App.UI.Settings;
+using NexusMods.App.UI.Controls.MarkdownRenderer;
 using ReactiveUI;
-using ReactiveUI.Fody.Helpers;
 
 namespace NexusMods.App.UI.Overlays.MetricsOptIn;
 
 /// <summary>
 /// Primary view model for the MetricsOptIn overlay.
 /// </summary>
+[UsedImplicitly]
 public class MetricsOptInViewModel : AOverlayViewModel<IMetricsOptInViewModel>, IMetricsOptInViewModel
 {
     private readonly ISettingsManager _settingsManager;
 
     public ICommand Allow { get; }
     public ICommand Deny { get; }
 
+    public IMarkdownRendererViewModel MarkdownRendererViewModel { get; }
+
     /// <summary>
     /// DI Constructor
     /// </summary>
-    public MetricsOptInViewModel(ISettingsManager settingsManager)
+    public MetricsOptInViewModel(ISettingsManager settingsManager, IMarkdownRendererViewModel markdownRendererViewModel)
     {
         _settingsManager = settingsManager;
+        MarkdownRendererViewModel = markdownRendererViewModel;
+
+        MarkdownRendererViewModel.Contents = """
+## Diagnostics and usage data
+
+Help us provide you with the best modding experience.
+
+With your permission, we will collect analytics information and sent it to our team to help us improve quality and performance.
```

Comment:
```suggestion
With your permission, we will collect analytics information and send it to our team to help us improve quality and performance.
```

Reply:
![2024-06-03_12-22-53](https://github.com/Nexus-Mods/NexusMods.App/assets/16577545/ba2659f8-2e5e-49b4-88be-445f74467bc0)

Reply:
You have to use `send` in the first sentence, and `sent` in the second sentence. 

```
With your permission, we will collect analytics information and send it to our team to help us improve quality and performance.

This information is sent anonymously and will never be shared with a 3rd party.
```

Reply:
Ah right.